### DISCUSSION TITLE
Dev mode - updates - what´s new?

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#6401] [General]
Hi dev team,

is it possible to create an overview in the discord channel, where the updates are described shortly? Espacially for the developer version :-p .... stable is decribed very well ....

Greets Klaus

**Comment by tracernz** [comment]
You can check https://github.com/flybywiresim/a32nx/blob/master/.github/CHANGELOG.md. Everything under the latest version number at the top is only in the dev version (will later be 0.8).
